README.md

```diff
@@ -83,7 +83,7 @@ Qubes uses SaltStack internally for VM provisionining and configuration manageme
 
 Installing this project is involved. It requires an up-to-date Qubes 4.1 installation running on a machine with at least 16GB of RAM (32 GB recommended).
 
-The project is currently in a closed beta, and we do not recommend installing it for production purposes. If you are participating in our beta program, please consult our *[end user documentation](https://workstation.securedrop.org)* for detailed setup instructions, and do not hesitate to reach out for assistance.
+The project is currently in a closed beta, and we do not recommend installing it indepependently for production purposes. If you are participating in our beta program, please consult our *[end user documentation](https://workstation.securedrop.org)* for detailed setup instructions, and do not hesitate to reach out for assistance. If you are interested in joining the beta, please contact us via securedrop@freedom.press or our support portal.
```

Comment:
[nit: indepependently -> independently]

I'm not sure about adding this line on joining the beta. Don't think we have sufficiently talked about rollout strategy to be ready for that.

Reply:
Yeah, I'm ambivalent as well. However, I think an email to us is probably better than an org or individual setting up the system without ever talking to us -- and it also creates opportunities for us to engage in needs assessment and provide advice.

Perhaps a middle ground could be:
- in the README, a more general "discuss your needs and learn more" invitation
- in the installer, a more explicit "we advise against installing without talking to us" warning.

What do you/others think?

Reply:
> and it also creates opportunities for us to engage in needs assessment and provide advice

I think there's a capacity question in there, that is I don't think we have enough to do _all the things_ at once. For that reason I'd prefer not adding any invitation.